.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directories:
+      - "/packages/http-client-python/eng/"
+      - "/packages/http-client-python/generator/test/azure"
+      - "/packages/http-client-python/generator/test/unbranded"
```

Comment:
Since there is also new added file `generate/test/dev_requirements.txt`, shall we add it here?